setup.py

```diff
@@ -63,6 +63,11 @@
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
```

Comment:
Also perhaps
```suggestion
```

If you want to remove deprecated py versions here too, to sync with the Travis config:
https://github.com/nats-io/nkeys.py/blob/299ce821aa73bfd55de48381fa9769faf8d2d621/.travis.yml#L3-L8